stdlib/public/core/SIMDVector.swift

```diff
@@ -1407,3 +1461,154 @@ where T: SIMD, T.Scalar: FloatingPoint {
   }
   return result
 }
+
+extension SIMD
+  where Self: Differentiable,
+        CotangentVector: SIMD,
+        Scalar : BinaryFloatingPoint,
+        /* Required in order to use unary negation operator due to following error:
+           >Self.CotangentVector.Scalar' does not conform to protocol 'FloatingPoint'
+        */
+        CotangentVector.Scalar: BinaryFloatingPoint {
+  @inlinable
+  static func _vjpAdd(
+    lhs: Self, rhs: Self
+    ) -> (Self, (CotangentVector) -> (CotangentVector, CotangentVector)) {
```

Comment:
Un-indent.

Reply:
Un-indent once so it's inline with `static` right?

Reply:
Yes